bot/cogs/rmq.py

```diff
@@ -0,0 +1,55 @@
+import logging
+
+import aio_pika
+from discord.ext.commands import AutoShardedBot
+
+from bot.constants import RabbitMQ
+
+
+log = logging.getLogger(__name__)
+
+
+class RMQ:
+    """
+    RabbitMQ event handling
+    """
+
+    rmq = None  # type: aio_pika.Connection
+    channel = None  # type: aio_pika.Channel
+    queue = None  # type: aio_pika.Queue
+
+    def __init__(self, bot: AutoShardedBot):
+        self.bot = bot
+
+    async def on_ready(self):
+        self.rmq = await aio_pika.connect_robust(
+            host=RabbitMQ.host, port=RabbitMQ.port, login=RabbitMQ.username, password=RabbitMQ.password
+        )
+
+        log.debug("Connected to RabbitMQ")
```

Comment:
In the previous implementation of this idea (websockets) the connection log was info.